backend/connector_processor/connector_processor.py

```diff
@@ -132,6 +134,8 @@ def test_connectors(connector_id: str, credentials: dict[str, Any]) -> bool:
             test_result = connector_impl.test_credentials()
             logger.info(f"{connector_id} test result: {test_result}")
             return test_result
+        except UnstractFSConnectorException as e:
+            raise UnstractFSException(detail=e.detail) from e
```

Comment:
@kirtimanmishrazipstack UnstractFSCOnnectorException is already a custom exception raised by us. Why are we again raising it as `UnstractFSException`?

Reply:
@ritwik-g since unstract/connector is a library it is producing `UnstractFSConnectorException` . All fs connector exception will inherit from this `UnstractFSConnectorException`
On backend  `UnstractFSException` will inherit django `APIException` and displays the error to FE

Reply:
NIT: @kirtimanmishrazipstack for brevity I suggest removing the `Unstract` prefix on `UnstractFSConnectorException` and renaming it as `FSConnectorError`

Reply:
done